yield_plugin/src/yield_plugin.cpp

```diff
@@ -427,11 +427,27 @@ namespace yield_plugin
     // Up until goal_pos (which also can be until end of the entire original trajectory), generate new speeds at
     // or near original trajectory points by generating them at a fixed time interval using the JMT polynomial equation
     const double initial_time = 0;
-    const double initial_accel = 0;
+    double initial_accel = 0;
+    if (last_speed_)
+    {
+      initial_accel = (initial_velocity - last_speed_.value()) /
```

Comment:
I am not sure at the moment whether if instantaneous acceleration I am using here is a good variable to use. Maybe low pass filter or some other way to approximate the ongoing acceleration is better?

Reply:
maybe we can get actual vehicle acceleration from vehicle status or something, but instantaneous is also reasonable